float64 input_num
---
float64 output_num